src/shared/watchdog.c

```diff
@@ -15,9 +15,9 @@
 #include "watchdog.h"
 
 static int watchdog_fd = -1;
-static char *watchdog_device = NULL;
-static usec_t watchdog_timeout = USEC_INFINITY;
-static usec_t watchdog_last_ping = USEC_INFINITY;
+static char *watchdog_device;
```

Comment:
why drop the NULL initialization here? it's gonna be NULL initialized anyway, so it might be worth making this explicit.

Reply:
because all static variables are always initialized with 0 so it's redundant and noisy. It's usually easier to spot quickly which static variable needs a special init value when only these variables are initialized.